### PR TITLE
Update service port naming inline with istio naming conventions

### PR DESCRIPTION
**Pull request by sravankumar777**
https://istio.io/docs/ops/deployment/requirements/

# Changes

To be on the safer side when deployed to a cluster with Istio deployed, we should follow service port naming convention as stated here -> https://istio.io/docs/setup/kubernetes/additional-setup/requirements/

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```
Named service ports: Service ports must be named. The port name key/value pairs must have the following syntax: name: <protocol>[-<suffix>]. See Protocol Selection for more details.
https://istio.io/docs/ops/deployment/requirements/
```

### CODE REVIEW

**Comment by sravankumar777** [comment]
@googlebot I fixed it.

**Comment by sravankumar777** [comment]
@googlebot I consent.

**Comment by sravankumar777** [comment]
@googlebot I fixed it.

**Comment by sravankumar777** [comment]
@googlebot I fixed it.

**Comment by sravankumar777** [comment]
@googlebot I fixed it.

**Comment by dibyom** [comment]
hi @sravankumar777 looks like the CLA is associated with the Github handle. Would you mind signing it with the email address associated with the commit in this PR?

**Comment by sravankumar777** [comment]
@googlebot I fixed it.

**Comment by sravankumar777** [comment]
@dibyom, i tried to do the CLA procedure with email address as well.
Not sure, how to fix this again.

**Comment by sravankumar777** [comment]
> @googlebot I fixed it.

**Comment by dibyom** [comment]
The email associated with the commit is `sravan.kumar@rakuten.com` Is that the one you used to sign the CLA with?

**Comment by sravankumar777** [comment]
@dibyom, thanks for your support. I have resigned CLA with correct email address in new commit.

File: config/400-webhook-service.yaml [hunk 1]
```diff
@@ -21,7 +21,8 @@ metadata:
   namespace: tekton-pipelines
 spec:
   ports:
-    - port: 443
+    - name: http-webhook
```
Comment (dibyom) [inline config/400-webhook-service.yaml]: Should this be https? https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/#manual-protocol-selection

Comment (sravankumar777) [reply config/400-webhook-service.yaml]: @dibyom, new commit added for change of ```http-webhook``` to ```https-webhook```.

Comment (dibyom) [reply config/400-webhook-service.yaml]: Nice!

**Comment by sravankumar777** [comment]
> Would you mind squashing the two commits into one? LGTM otherwise!

Thanks for your comments.
I did squash the commits, please check it.

**Comment by chmouel** [comment]
/lgtm

Looking Good! 🤙🏽 I would love to see a use case on how istio and tekton are used on the control plane..

**Pull request merged**